eras/babbage/impl/src/Cardano/Ledger/Babbage/Rules/Ledger.hs

```diff
@@ -14,12 +14,11 @@
 module Cardano.Ledger.Babbage.Rules.Ledger (BabbageLEDGER) where
 
 import Cardano.Ledger.Alonzo.Rules (AlonzoUtxowEvent, ledgerTransition)
-import Cardano.Ledger.Alonzo.Tx (AlonzoEraTx)
+import Cardano.Ledger.Alonzo.Tx
```

Comment:
did you mean to switch to open imports? I was thinking that just `Core` modules were exempt from the open import ban..

Reply:
Nah, it was unintentional